app/src/main/java/com/example/cocktaildictionary/viewmodel/HomeActivityViewModel.kt

```diff
@@ -47,6 +85,7 @@ class HomeActivityViewModel(application: Application): AndroidViewModel(applicat
          cacheDatabase.cacheDao().addToCache(cache)
       }
       Log.d(TAG,"Successfully added to cache")
+      timer.start()
```

Comment:
What happens when you close/kill the app and reopen it, does the timer kick off again?

Reply:
Yes another timer kicks off whilst the initial timer continues to countdown

Reply:
Sounds like I can trick the app by constantly killing it and opening it. But cool. Can also be explored further later on.